src/main/java/net/sf/mzmine/parameters/parametertypes/filenames/FileNameParameter.java

```diff
@@ -99,7 +99,9 @@ public FileNameParameter cloneParameter() {
     public void setValueFromComponent(FileNameComponent component) {
         File compValue = component.getValue();
         if (extension != null) {
-            if (!compValue.getName().endsWith(extension))
+            if (!(compValue.getName().toUpperCase()
+                    .endsWith(extension.toUpperCase()) || compValue.getName()
+                    .toLowerCase().endsWith(extension.toLowerCase())))
```

Comment:
I see. Well actually I do find the the third case a bit awkward. I would prefer to just check for the existence of the extension, and ignore the case completely.